kolibri/auth/test/test_permissions.py

```diff
@@ -1,454 +1,454 @@
-from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
Commented out the tests in this file for now -- I'll be writing tests from scratch, but useful to have this as a reference until that's done.